app/templates/play/modal/amazon-hoc-modal.jade

```diff
@@ -31,3 +31,6 @@
         a#aws-future-eng-link(href="http://amazonfutureengineer.com/" data-i18n="amazon_hoc.future_eng_4" target="_blank")
         =" "
         span(data-i18n="amazon_hoc.future_eng_5")
+      
+      if(view.options.showContinue)
+        button.btn.btn-illustrated.btn-success.btn-lg#continue-button Continue Playing
```

Comment:
Let's data-i18n this button text for consistency.  There's already a courses.continue_playing string in en.coffee that you can reuse.